README.md

```diff
@@ -365,6 +365,19 @@ The format of the JSON is a a dictionary with each key/value pair mapping to the
 }
 ```
 
+# Stelligent Policy Complexity Metrics (spcm)
+
+The basis for SPCM is describe in the blog post: https://stelligent.com/2020/03/27/thought-experiment-proposed-complexity-metric-for-iam-policy-documents/
+
+Starting in version 0.6.0 of cfn_nag:
+* `spcm_scan` can scan a directory of CloudFormation templates (like cfn_nag_scan) and generate a report with the SPCM
+   metrics in either JSON or HTML format
+* A rule is added to warn on an IAM::Policy or IAM::Role with a SPCM score of >= 50
```

Comment:
Since this is configurable, we can state here that >= 50 is just the default recommendation.